### PR TITLE
friendly URL containing umlaut was never found. proposing umlaut over…

### PR DESCRIPTION
**Pull request by mheimschild**
…write when creating new page

### CODE REVIEW

File: packages/plugin-packages/mashroom-portal/src/backend/utils/model_utils.ts [hunk 1]
```diff
@@ -58,7 +58,7 @@ export const getPageData = async (sitePath: string, friendlyUrl: string | undefi
 
     let pageRef = null;
     if (friendlyUrl) {
-        pageRef = await portalService.findPageRefByFriendlyUrl(site, friendlyUrl);
+        pageRef = await portalService.findPageRefByFriendlyUrl(site, decodeURIComponent(friendlyUrl));
```
Comment (nonblocking) [inline packages/plugin-packages/mashroom-portal/src/backend/utils/model_utils.ts:61]: Is this really the right place to do the decoding of the friendlyUrl? like this the same problem could occur in other places; 

this should be done in renderPortalPage()

File: packages/plugin-packages/mashroom-portal-admin-app/src/js/components/PageConfigureDialog.tsx [hunk 2]
```diff
@@ -241,7 +241,13 @@ export default class PageConfigureDialog extends PureComponent<Props> {
             const previousTitle = typeof (previousValues.page.title) === 'object' ? previousValues.page.title[this.props.languages.default] : previousValues.page.title;
 
             if (title && title !== previousTitle) {
-                let friendlyUrl = title.replace(/[ -]/g, '_').toLowerCase();
+                let friendlyUrl = title
```
Comment (nonblocking) [inline packages/plugin-packages/mashroom-portal-admin-app/src/js/components/PageConfigureDialog.tsx:244]: I don't like it; it overrides the user input without any warning or hint; also, what about all the special characters of other languages?

**Review by nonblocking — CHANGES_REQUESTED** [review]

File: packages/plugin-packages/mashroom-portal-admin-app/src/js/components/PageConfigureDialog.tsx (hunk 2 above)
Comment (mheimschild) [reply packages/plugin-packages/mashroom-portal-admin-app/src/js/components/PageConfigureDialog.tsx:244]: That's true. URLs with special characters are perfect valid but not very common.
Here is a more generic solution needed for language using non ascii chars to not force users to correct all friendlyUrls derived from the page title.
Rollbacking current change

**Comment by nonblocking** [comment]
Should be fixed with https://github.com/nonblocking/mashroom/commit/f2d44ce668341cffc406250f6a2eea9631ed6fa0

**Pull request closed without merging**